interface/patient_file/summary/demographics_save.php

```diff
@@ -123,8 +124,8 @@
     filter_input(INPUT_POST, 'i2policy_type')
 );
 
-$i3dob  = fixDate(filter_input(INPUT_POST, "i3subscriber_DOB"));
-$i3date = fixDate(filter_input(INPUT_POST, "i3effective_date"), date('Y-m-d'));
+$i3dob  = DateToYYYYMMDD(filter_input(INPUT_POST, "i3subscriber_DOB"));
+$i3date = empty(filter_input(INPUT_POST, "i3effective_date")) ? oeFormatShortDate(date('Y-m-d')) : filter_input(INPUT_POST, "i3effective_date");
```

Comment:
TODO:
Look into why oeFormatShortDate is being used in a save script.

Reply:
I don't know why, but in edit insurance form (in the demographics) in the old code if 'effective_date' field is empty the default in the fixDate function is  date('Y-m-d'), so I tried to save on this behavior..

Reply:
Fixed this to mimick the above new_comprehensive_save.php script changes.

Reply:
DONE

Reply:
script good to go